src/scripts/components/VariableMessage.jsx

```diff
@@ -2,8 +2,18 @@ import cx from 'classnames';
 import PropTypes from 'prop-types';
 import React, { Fragment } from 'react';
 import ReactDOM from 'react-dom';
-
-import { Button, FormLabel, FormExplanationMessage, FormValidationMessage, Message, ToggleButton, UtilitySystem } from '../components';
+import { Button,
+  FormLabel,
+  FormExplanationMessage,
+  FormValidationMessage,
+  Message,
+  ToggleButton,
+  UtilitySystem,
+} from '../components';
+
+const BACKSPACE_KEY = 8;
```

Comment:
I know I approved already, but responding to the point @alexgsummer brought up.  If we are making this change, does it make sense to create a constants file in rhinostyle? I could see these being sharable amongst other components in the future.

Reply:
Makes sense to me. `constants.js` file added to `/src`. Let me know if you'd like it to live somewhere else.